src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -144,28 +145,30 @@ def unregister_stale(self, current_epoch):
             self.unregister(pool_id, namespace)
 
 
+ImageSpecT = Tuple[str, str, str]
```

Comment:
as far as I can see, this module desperately cries for a 

```python
class ImageSpec(NamedTuple):
    pool_id: str,
    namespace: str
    image_id: str
```

Reply:
done.